### PR TITLE
[autoscaler] rsync bootstrap flag

### PR DESCRIPTION
**Pull request by DmitriGekhtman**
## Why are these changes needed?

Provides a flag to turn off config bootstrap for rsync in situations where the config is already bootstrapped and re-bootstrapping might have unintended consequences. Main intended application is syncing results from workers to head in Ray Tune.

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/autoscaler/_private/commands.py [hunk 1]
```diff
@@ -989,6 +989,7 @@ def rsync(config_file: str,
           use_internal_ip: bool = False,
           no_config_cache: bool = False,
           all_nodes: bool = False,
+          do_config_bootstrap: bool = True,
```
Comment (richardliaw) [inline python/ray/autoscaler/_private/commands.py:992]: Can you rename to `should_bootstrap`?

**Review by richardliaw — APPROVED** [review]

**Review by ijrsvt — APPROVED** [review]
LGTM!

**Comment by DmitriGekhtman** [comment]
This is good to merge.

**Comment by ijrsvt** [comment]
Test Scheduling is known to be flaky.

**Pull request merged**